main.py

```diff
@@ -0,0 +1,36 @@
+from pyspark.sql import SparkSession
+
+if __name__ == "__main__":
+    spark = SparkSession.builder \
+        .master("local[*]") \
+        .appName("Films") \
+        .getOrCreate()
+
+    t_basics = spark.read.option("sep", "\t") \
+        .csv("data/t_basics.tsv", inferSchema=True, header=True)
+    ratings = spark.read.option("sep", "\t") \
+        .csv("data/ratings.tsv", inferSchema=True, header=True)
+
+    t_basics.join(ratings, on=["tconst"]) \
+        .filter((t_basics.titleType == "movie")
+                    & (ratings.numVotes >= 100000)) \
```

Comment:
you do the same `filter`  in each block of code, maybe it's better to do it only once?